wire-ios-sync-engine/Source/UserSession/ZMUserSession/ZMUserSession.swift

```diff
@@ -1039,6 +1038,15 @@ extension ZMUserSession: ZMSyncStateDelegate {
         guard e2eiFeature.isEnabled else { return }
         NotificationCenter.default.post(name: .checkForE2EICertificateExpiryStatus, object: nil)
     }
+
+    // MARK: - Observers
+
+    private func observeMLSGroupVerificationStatus() {
+        guard e2eiFeature.isEnabled else { return }
```

Comment:
Don't we waste energy with this check. It looks like a simple property, but in fact it uses the feature repository to access core data. Can it really change during the runtime of the app?

Reply:
removed